### PR TITLE
port purchase_order_revision to 8.0

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by eLBati** [comment]
thanks @gurneyalex please note the file `i18n/de.po` is present in 7.0 version but missing in `__unported__`. You should add it again

**Comment by gurneyalex** [comment]
@elBati @rvalyi your feedback on this PR is very welcome, as I made some enhancements on the 7.0 version. See also https://github.com/OCA/sale-workflow/pull/137

File: purchase_order_revision/i18n/en.po [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline purchase_order_revision/i18n/en.po]: This file is not needed.

File: purchase_order_revision/README.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Revisions for purchase orders (and requests for quotation)
+==========================================================
+
+On canceled orders, you can click on "New revision" button. This
+will create a new revision of the quotation, with the same base number and a
+'-revno' suffix appended. A message is added in the chatter saying that a new
+revision was created.
+
+In the form view, a new tab is added that lists the previous revisions, with
+the date of their were made obsolete and the user who performed the action.
```
Comment (pedrobaeza) [inline purchase_order_revision/README.rst]: s/the date of their were made/the date when it was made

**Comment by pedrobaeza** [comment]
Little remarks, but the rest seems OK

**Comment by eLBati** [comment]
Just the same remark as
https://github.com/OCA/sale-workflow/pull/137#discussion_r28222386
but LGTM :+1:

**Comment by gurneyalex** [comment]
@pedrobaeza @eLBati updated the PR with your remarks.

File: purchase_order_revision/__openerp__.py [hunk 3]
```diff
@@ -21,24 +21,16 @@
 
 {
     'name': "Purchase order revisions",
-    'version': '0.1',
+    'version': '0.2',
```
Comment (pedrobaeza) [inline purchase_order_revision/__openerp__.py]: Bump to version 1.0. It deserves it :wink:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by eLBati** [comment]
Thanks :+1:

**Pull request merged**